soundcork/main.py

```diff
@@ -245,3 +248,14 @@ def bose_xml_response(xml: ET.Element, etag: int = 0, method: str = "") -> Respo
     response.headers["etag"] = str(etag)
     response.headers["method_name"] = method
     return response
+
+
+def validate_params(account=12345, device="ABCD3"):
```

Comment:
I'm fine with this as a lil hack, but I'd rather see `if account; try int(account)` etc.